packages/components/src/Components/NotAuthorized/notAuthorized.scss

```diff
@@ -0,0 +1,19 @@
+@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all';
+
+.ins-c-component_not-authorized {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+
+    .ins-c-text__sorry { max-width: 70%; }
+
+    h1, svg, h2 { @include rem('margin-bottom', 30px); }
```

Comment:
Either use `var(--pf-global--spacer--lg)` or `var(--pf-global--spacer--xl)` depends on what fits it better.